distribution/zip/ballerina-tools/pom.xml

```diff
@@ -138,6 +138,12 @@
             <type>zip</type>
             <classifier>ballerina-binary-repo</classifier>
         </dependency>
+        <dependency>
```

Comment:
you don't need to put this in both ballerina distribution and ballerina-tools distribution, if you add that to ballerina distribution, it will automatically comes to ballerina-tools distribution